docs/decisions/0001-course-experience.rst

```diff
@@ -0,0 +1,156 @@
+Courses units in LMS
+--------------
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+NELP business case requires multiples features that are not part
+of open-edx platform core, and it's necessary a new implementation
+that allows users to report and share their opinions about the
+course content and experience during the course.
+
+Decisions
+=========
+
+1. Create a course experience module.
+2. Create ``Like`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
+
+  * status => String field with choices: Liked, Unliked and not-set.
+
+  * usage_id => String field, this is course subsection identifier.
+
+3. Create ``Report`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
+
+  * reason => String field with choices:
+
+   - Sexual content
+
+   - Graphic violence.
+
+   - Hateful or abusive content.
+
+   - Copycat or impersonation.
+
+   - Other objection.
+
+  * usage_id => String field, this is course subsection identifier.
+
+4. Set model constrains, ``Like`` and ``Report`` model should be unique per sub-sections and user.
+5. Create API views for each model.
+
+
+API Examples
+~~~~~~~~~~~~
+  GET request
+
+  /eox-nelp/api/experience/v1/<usage-id>/rate
```

Comment:
That has to be in that url  shape?.  `api/experience/v1`  or could be like other endpoints: `experience/v1/`: 
![image](https://user-images.githubusercontent.com/51926076/220194625-b2775a8d-f9db-429b-9be0-75dcb511ee52.png)

 by the way, mfe config API that is "new " has that shape. api/mfe_config/v1/

Reply:
@johanv26 please expose arguments

Reply:
no, I am only asking what is the consideration to choose that shape of the API URL?

Reply:
commonly the word `api` is used to make a difference between view endpoints and endpoints that returns just json data, in django is very common to have  both those views, so this is a good way to identify each one, however this is not an standard or hard rule